### ISSUE TITLE
Compatibility with TYPO3 v11

### ISSUE THREAD

**Issue by brotkrueml** [b13/link2language#7]
Using this extension is very helpful. As we are now upgrading our website to TYPO3 v11, it would be nice to have the extension compatible with it.

I digged a little bit into it. The setting of the template root path must be muved from the initialize() method to the render method(), as `$this->view` is not available at this place. However, also the `Page.html` template changed a lot.

I see two possibilities:
- Set the template root path dependend on the TYPO3 version and deliver two templates (before v11 and v11+).
- Make the extension compatible with v11 and publish a new version which only depends on v11+.
 
For a clean approach I would go for the second variant and will try to add a PR, if this is okay.

**Comment by bmack** [comment]
Hey @brotkrueml ,

we're very much in favor of supporting as many TYPO3 Core versions as possible at the same time, so I'm in favor of adding a new PHP class for v11 which has different styles and implementation.

**Comment by bmack** [comment]
Interested in checking the PR?

**Comment by brotkrueml** [comment]
Yup, will do.
